logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/Message.kt

```diff
@@ -8,10 +8,10 @@ data class Message(
     val id: String,
     val content: MessageContent,
     val conversationId: ConversationId,
-    val date: String,
-    val senderUserId: UserId,
-    val senderClientId: ClientId,
-    val status: Status
+    val date: String = "",
```

Comment:
Why not take the current time as the default?

Reply:
Data class should not be ommiting stuff. Date is not optional, it's mandatory.